src/components/Person/Person.jsx

```diff
@@ -1 +1,44 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
```

Comment:
```suggestion
     {`My name is ${name}`}
```
Consider using template strings here